conf/application.conf

```diff
@@ -47,38 +47,43 @@ play.http.errorHandler = "handlers.ErrorHandler"
 
 
 microservice {
-  services {
-      auth {
-        protocol = http
-        host     = localhost
-        port     = 8500
-      }
-      emcs-tfe-change-destination-frontend {
-        protocol = http
-        host = localhost
-        port = 8319
-      }
-      emcs-tfe {
-        protocol = http
-        host = localhost
-        port = 8311
-      }
-      user-allow-list {
-        protocol = http
-        host = localhost
-        port = 11306
-      }
-      reference-data {
-        protocol = http
-        host = localhost
-        port = 8312
-      }
+    services {
+        auth {
+            protocol = http
+            host = localhost
+            port = 8500
+        }
+        emcs-tfe-change-destination-frontend {
+            protocol = http
+            host = localhost
+            port = 8319
+        }
+        emcs-tfe {
+            protocol = http
+            host = localhost
+            port = 8311
+        }
+        user-allow-list {
+            protocol = http
+            host = localhost
+            port = 11306
+        }
+        emcs-tfe-reference-data {
+            protocol = http
+            host = localhost
+            port = 8312
+        }
+        emcs-tfe-reference-data-stub {
+            protocol = http
+            host = localhost
+            port = 8309
+        }
         contact-frontend {
-          protocol = http
-          host = localhost
-          port = 9250
+            protocol = http
+            host = localhost
+            port = 9250
         }
-  }
+    }
 }
```

Comment:
Could you create (or link me to) the app-config-* PR's for these entries, please?